clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -874,9 +874,13 @@ public void subscribe(Pattern pattern, ConsumerRebalanceListener listener) {
     public void unsubscribe() {
         acquire();
         try {
+            //TODO: Reactivate the commented code once KAFKA-3491 is resolved
```

Comment:
Instead of commenting these sections out, could we open a JIRA to make sure we come back to it?